### PR TITLE
Contact form branch

### PR DESCRIPTION
**Pull request by sameh080081**
this branch is for the contact page created by two partners.

### CODE REVIEW

File: mobile.html [hunk 1]
```diff
@@ -156,5 +156,16 @@ <h5>2015</h5>
             <li><img src="./images/icons/Separator.png" alt="sep"></li>
         </ul>
     </section>
+    <div class="form-section">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form class ="form" action="https://formspree.io/f/mgedlyee" method="post">
+            <input class="input1" type="text" placeholder="Yeremias" maxlength="30">
+            <input class="input1" type="email" placeholder="natajayanj14@gmail.com">
+            <textarea class="message" name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here"></textarea>
```
Comment (Deepakdanger) [inline mobile.html]: - [x] All input fields have the `required` attribute to activate the validation on submit. Let's add `required in every input.

File: mobile.html [hunk 2]
```diff
@@ -156,5 +156,16 @@ <h5>2015</h5>
             <li><img src="./images/icons/Separator.png" alt="sep"></li>
         </ul>
     </section>
+    <div class="form-section">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form class ="form" action="https://formspree.io/f/mgedlyee" method="post">
+            <input class="input1" type="text" placeholder="Yeremias" maxlength="30">
+            <input class="input1" type="email" placeholder="natajayanj14@gmail.com">
```
Comment (Deepakdanger) [inline mobile.html]: - [x] In `Contact form` section, In input `name and email` placeholder ,kindly use placeholder text like ` Full name` & `E-mail` rather then value.

File: mobile.html [hunk 3]
```diff
@@ -156,5 +156,16 @@ <h5>2015</h5>
...
+            <textarea class="message" name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here"></textarea>
+            <button class="submit">Get It Touch</button>
```
Comment (Deepakdanger) [inline mobile.html]: - [x] The submit button can be either an `input` element with the type `submit` or a `button` element with the type `submit`. Please write `type= "submit"` .

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @sameh080081, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: mobile.html (hunk 2 above)
Comment (sameh080081) [reply mobile.html]: actually, I was confused about that because there is a value in the template, so I did it exactly like the template.

**Review by FrederickMih — APPROVED** [review]
# Status : Approved :tada: :tada: 

Hi @sameh080081 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**